2.14/2.14.4/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.14.4
+Сообщение первое: "Введите первое число от 0 до 40" (например ввели 12)
+Сообщение второе: "Введите второе число от 0 до 40" (например ввели 21)
+Потом появляется алерт: "Сумма этих чисел равна тридцать три" (для нашего примера получился ответ 33)
+Специфика в том, чтобы в алерте выводить числа текстом.
+Предусмотреть текст для чисел в диапазоне 0 - 99.
+<script>
```

Comment:
check all your variables and think which ones should be declared by ˋconstˋ and which ones by ˋletˋ

Reply:
Just check )) perhups everything is ok ))